README.md

```diff
@@ -16,20 +16,20 @@ Zeebe provides visibility into and control over business processes that span mul
 * Export process data for monitoring and analysis
 * Engage with an active community
 
-[Learn more at zeebe.io](https://zeebe.io)
+[Learn more at camunda.com](https://camunda.com/products/cloud/workflow-engine/)
 
 ## Status
 
-Starting with Zeebe 0.20.0, the "developer preview" label was removed from Zeebe and the first production-ready version was released  .
+We are currently working on Zeebe 1.0.
 
-To learn more about what we're currently working on, please visit the [roadmap](https://zeebe.io/roadmap).
+To learn more about what we're currently working on, check the [GitHub issues](https://github.com/camunda-cloud/zeebe/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) and the [latest commits](https://github.com/camunda-cloud/zeebe/commits/develop).
 
 ## Helpful Links
 
-* [Blog](https://zeebe.io/blog)
+* [Blog](https://camunda.com/blog/category/process-automation-as-a-service/)
```

Comment:
Is PaaS the best category? I think https://camunda.com/blog/category/camunda-cloud/ is also a valid option, as well as https://camunda.com/?post_type=post&s=zeebe

Reply:
This is where the https://zeebe.io/blog redirect targets, so I used that. We can ask them to change the redirect is another category is appropriate.

Reply:
Wow, each of those categories looks like it could be the thing....

Reply:
Heard back from marketing - that's the correct link to use.